### PR TITLE
v0.68.0 — - Updated heading font weight to match latest UI designs.

### PR DESCRIPTION
**Pull request by DamianMullins**
### Changed
- Updated heading font weight to match latest UI designs.

### CODE REVIEW

**Comment by DamianMullins** [comment]
To go in after #121

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "@justeat/fozzie",
   "title": "Fozzie – Just Eat UI Web Framework",
   "description": "UI Web Framework for the Just Eat Global Platform",
-  "version": "0.66.0",
+  "version": "0.67.0",
```
Comment (sonnyprince) [inline package.json]: Inconsistent package / changelog version numbers :)

Comment (DamianMullins) [reply package.json]: 😱

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,15 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+
+v0.68.0
+------------------------------
+*August 30, 2018*
+
+### Changed
+- Updated heading font size to match latest UI designs.
```
Comment (ashleynolan) [inline CHANGELOG.md]: Looks like a font-weight is being changed…

Comment (DamianMullins) [reply CHANGELOG.md]: Oh $*^%! Will update now 😄

**Comment by DamianMullins** [comment]
I screwed this one up a little; it's waiting behind #121. I've recreated the PR in #125

**Pull request merged**